src/js/loading-spinner.js

```diff
@@ -34,6 +34,22 @@ class LoadingSpinner extends Component {
 
     return el;
   }
+
+  /**
+   * Show the `LoadingSpinner`s DOM element, by adding the `show` css class to it.
+   */
+  show() {
+    super.show();
+    this.el().classList.add('show');
```

Comment:
Component has an API for adding and removing classes [`this.addClass`](https://github.com/videojs/video.js/blob/master/src/js/component.js#L721)

Reply:
Changed in latest PR